package/yast2-auth-server.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Jul 27 00:50:39 UTC 2022 - William Brown <william.brown@suse.com>
+
+- Version 4.6.0
```

Comment:
As I commented in https://github.com/yast/yast-auth-client/pull/105

> 
> 
> I have checked the changes files and it is following the versioning schema proposed at https://yastgithubio.readthedocs.io/en/latest/versioning/
> 
> Thus, I'd use 4.5.1 instead of 4.6.0.